src/filters/permalink-filter.js

```diff
@@ -0,0 +1,9 @@
+// Permalink possible fix from https://github.com/11ty/eleventy/issues/923
+//   "permalink": "{{ page | post_permalink }}"
+
+module.exports = page => {
+  const yyyy = page.date.getFullYear();
+  const mm = String(page.date.getMonth() + 1).padStart(2, "0");
+  const slug = page.fileSlug.replace(/^\d{4}-\d{2}-\d{2}\./, "");
+  return `${yyyy}/${mm}/${slug}/`;
+};
```

Comment:
So, we just moved the function from L33-38 of .eleventy.js and moved it into this file.
Note that we're only exporting an anonymous function which takes one parameter still (the `page` object), and we'll infer the page's `date`, and `fileSlug` from that and then return a permalink string to our template. The code here is all the same, just copy/pasted into a ./src/filters/*.js file to be consistent with your other filters.